languages/org.xtext.builddsl/src-gen/org/xtext/builddsl/services/BuildDSLGrammarAccess.java

```diff
@@ -448,13 +446,7 @@ public ParserRule getXOtherOperatorExpressionRule() {
 	}
 	
 	//OpOther:
-	//	'->'
```

Comment:
wonder why this has changed

Reply:
i dont get these changes but changes to generated xtend code

Reply:
Do you mean that one line or the entire comment change?

Reply:
dont get any grammar access changes